udata_hydra/routes/__init__.py

```diff
@@ -0,0 +1,27 @@
+from aiohttp import web
+
+from udata_hydra.routes.checks import get_all_checks, get_latest_check
+from udata_hydra.routes.resources import create_resource, delete_resource, update_resource
+from udata_hydra.routes.status import get_crawler_status, get_stats, get_worker_status
+
+# routes = web.RouteTableDef()
```

Comment:
Is that a leftover comment that should be removed?

Reply:
It's on purpose, as a reminder of how aiohttp routes can be typed... aiohttp doc is pretty bad so I decided to keep that as a uselful comment.